environments/staging.yaml

```diff
@@ -3,4 +3,12 @@ continuumLogin:
 
 continuumAdaptor:
   port: 3001
-  continuumApiUrl: "http://continuumtest--gateway.elife.internal"
\ No newline at end of file
+  continuumApiUrl: "http://continuumtest--gateway.elife.internal"
+
+submission:
+  scienceBeam:
+    apiUrl: "https://sciencebeam-texture.elifesciences.org/api/convert"
+  aws:
+    accessKeyId: ""
```

Comment:
comment out these keys if they are not really used? A default should be in `values.yml`, if there is any